rmf_fleet_adapter/src/rmf_fleet_adapter/events/GoToPlace.cpp

```diff
@@ -745,12 +746,15 @@ void GoToPlace::Active::_execute_plan(
   }
   else
   {
+    auto event = rmf_task::events::SimpleEventState::make(_assign_id->assign(), "detour", "", Status::Underway, {});
```

Comment:
Instead of a blank string for the description, maybe we can put something like `"the robot is parking until its destination becomes available"`.